### PR TITLE
Simplify tradable exits

### PR DESCRIPTION
**Pull request by hkalodner**
Previously we used a combination of node hash and index within that node to identify sends. Now that we're tracking the total number of messages sent from a given chain, we can uniquely identify sends just based off of their index.

This both simplifies the tradable exits implementation and means that you can use tradable exits before an assertion has been made containing your withdrawal.

### CODE REVIEW

**Comment by hkalodner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 8
           

Complexity increasing per file
==============================
- packages/arb-validator/chainobserver/ethbridge_test.go  10
         

Clones added
============
- packages/arb-validator/chainobserver/ethbridge_test.go  2
- packages/arb-validator-core/ethbridgetestcontracts/rolluptester.go  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5985749&bid=19628542)

File: packages/arb-validator-core/valprotocol/confirmation.go [hunk 1]
```diff
@@ -21,68 +21,62 @@ import (
 	"math/big"
 )
 
-type ConfirmValidOpportunity struct {
+type ConfirmNodeOpp struct {
```
Comment (edfelten) [inline packages/arb-validator-core/valprotocol/confirmation.go:24]: I understand the desire to shorten the name, but "Opp" seems too ambiguous.  I could be "opposite" or "opposition".  Suggest Oppty, with a comment at the definition (or just stick with Opportunity.

**Comment by hkalodner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 8
           

Complexity increasing per file
==============================
- packages/arb-validator/chainobserver/ethbridge_test.go  10
         

Clones added
============
- packages/arb-validator-core/ethbridgetestcontracts/rolluptester.go  8
- packages/arb-validator/chainobserver/ethbridge_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5985749&bid=19628542)

File: packages/arb-validator-core/valprotocol/confirmation.go (hunk 1 above)
Comment (hkalodner) [reply packages/arb-validator-core/valprotocol/confirmation.go]: Updated

**Comment by hkalodner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 8
           

Complexity increasing per file
==============================
- packages/arb-validator/chainobserver/ethbridge_test.go  10
         

Clones added
============
- packages/arb-validator/chainobserver/ethbridge_test.go  2
- packages/arb-validator-core/ethbridgetestcontracts/rolluptester.go  8
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5985749&bid=19628542)

**Comment by hkalodner** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 8
- Added 8
           

Complexity increasing per file
==============================
- packages/arb-validator/chainobserver/ethbridge_test.go  10
         

Clones added
============
- packages/arb-validator-core/ethbridgetestcontracts/rolluptester.go  8
- packages/arb-validator/chainobserver/ethbridge_test.go  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5985749&bid=19628542)

**Pull request merged**